Addon loader doesn't work in Electron

Since the 2.0 release the loadAddon method doesn't work anymore in Electron apps.

Code:
```
var Terminal = require('xterm');
Terminal.loadAddon('fit');
```
Error:
`Uncaught Error: Cannot find module '/src/../addons/fit'`

The same code does work with version 1.1.3.

## Details
- Browser and browser version: Electron 1.4.3 (Chrome 53.0.2785.113)
- OS version: MacOS Sierra
- xterm.js version: 2.0.0 / 2.0.1

--- a/addons/attach/attach.js
+++ b/addons/attach/attach.js
@@ -11,12 +11,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = attach(require('../../src/xterm'));
+    module.exports = attach(require('../../dist/xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../src/xterm'], attach);
+    define(['../../dist/xterm'], attach);
   } else {
     /*
      * Plain browser environment

--- a/addons/fit/fit.js
+++ b/addons/fit/fit.js
@@ -16,12 +16,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = fit(require('../../src/xterm'));
+    module.exports = fit(require('../../dist/xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../src/xterm'], fit);
+    define(['../../dist/xterm'], fit);
   } else {
     /*
      * Plain browser environment

--- a/addons/fullscreen/fullscreen.js
+++ b/addons/fullscreen/fullscreen.js
@@ -15,12 +15,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = fullscreen(require('../../src/xterm'));
+    module.exports = fullscreen(require('../../dist/xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../src/xterm'], fullscreen);
+    define(['../../dist/xterm'], fullscreen);
   } else {
     /*
      * Plain browser environment

--- a/addons/linkify/linkify.js
+++ b/addons/linkify/linkify.js
@@ -3,12 +3,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = linkify(require('../../src/xterm'));
+    module.exports = linkify(require('../../dist/xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../src/xterm'], linkify);
+    define(['../../dist/xterm'], linkify);
   } else {
     /*
      * Plain browser environment

--- a/src/xterm.js
+++ b/src/xterm.js
@@ -642,7 +642,7 @@ Terminal.prototype.open = function(parent) {
 Terminal.loadAddon = function(addon, callback) {
   if (typeof exports === 'object' && typeof module === 'object') {
     // CommonJS
-    return require(__dirname + '/../addons/' + addon);
+    return require('../addons/' + addon);
   } else if (typeof define == 'function') {
     // RequireJS
     return require(['../addons/' + addon + '/' + addon], callback);

--- a/test/addons/linkify-test.js
+++ b/test/addons/linkify-test.js
@@ -1,5 +1,5 @@
 var assert = require('chai').assert;
-var Terminal = require('../../src/xterm');
+var Terminal = require('../../dist/xterm');
 var linkify = require('../../addons/linkify/linkify');
 
 describe('linkify addon', function () {

--- a/test/addons/test.js
+++ b/test/addons/test.js
@@ -1,10 +1,11 @@
 var assert = require('chai').assert;
 var Terminal = require('../../src/xterm');
+var distTerminal = require('../../dist/xterm');
 
 describe('xterm.js addons', function() {
   it('should load addons with Terminal.loadAddon', function () {
     Terminal.loadAddon('attach');
-    // Test that function was loaded successfully
-    assert.equal(typeof Terminal.prototype.attach, 'function');
+    // Test that addon was loaded successfully
+    assert.equal(typeof distTerminal.prototype.attach, 'function');
   });
 });

